x-pack/plugins/observability/public/pages/alerts/alerts.tsx

```diff
@@ -176,6 +178,9 @@ function InternalAlertsPage() {
         }}
       >
         <EuiFlexGroup direction="column" gutterSize="m">
+          <EuiFlexItem>
+            <MaintenanceWindowCallout kibanaServices={kibanaServices} />
```

Comment:
Should we test this too?

Reply:
I thought really long time for it.
That's my conclusion. The MaintenanceWindowCallout is covered with unit test. So it's good with this part.
Here I think max we need to test that no one deleted MaintenanceWindowCallout component from Alert table. It can be done in unit test(like to check if we have this component in DOM tree), but they do not have it. They do not have it. (I did not find proper functional test for the whole thing as well). So I decided to stick with what it is.

Reply:
I think we are owners of maintenance window, and as such, adding a test here would enhance our confidence to ensure that even during potential refactoring by the o11y team, our call out remains visible. If there’s no existing test suite that aligns perfectly due to different testing focuses or specific mocks, perhaps we could consider creating a dedicated testing space for this. Would that be feasible?

Reply:
> I did not find proper functional test for the whole thing as well

@guskovaue  Have you checked the [test directory that I shared](https://elastic.slack.com/archives/C023GDA0WMP/p1691668062927609?thread_ts=1691666610.406089&cid=C023GDA0WMP)?
https://github.com/elastic/kibana/tree/main/x-pack/test/observability_functional/apps/observability/pages/alerts